checkov/cloudformation/runner.py

```diff
@@ -56,6 +56,9 @@ def run(
 
         if self.context is None or self.definitions is None or self.breadcrumbs is None:
             self.definitions, self.definitions_raw = create_definitions(root_folder, files, runner_filter, parsing_errors)
+            if self.definitions == {} and self.definitions_raw == {}:
+                logging.debug("Stopping CF Runner. Not 1 CF Template found.")
+                exit()
```

Comment:
this will fail on windows environments, because we don't multi process them, just multi thread. This will result in ending the complete process prematurely.